main.js

```diff
@@ -1,21 +1,37 @@
 const smilesAll = document.querySelectorAll(".smile");
 const counters = document.querySelectorAll(".counter");
+const voting = document.querySelectorAll(".voting");
```

Comment:
Plz, remove the unnecessary code and refactor

Your code may look like this, approximately
```
document.querySelectorAll(".voting").addEventListener("click", (event) => {
  if (event.target.classList.contain("smile")) {
    const counter = event.target.nextElementSibling;

    counter.innerText = Number(counter.textContent) + 1;
  }
})
```